### ISSUE TITLE
opam conflicts field accepts only conjunctions

### ISSUE THREAD

**Issue by edwintorok** [ocaml/oasis2opam#20]
The latest oasis2opam can generate correct conflicts field for most situations (when only a single version comparison is used), but if the version constraint is a conjunction then opam doesn't accept the generated constraint.

I'll send a new PR once I've [clarified the behaviour in opam](https://github.com/ocaml/opam/issues/2246)

**Comment by Chris00** [comment]
Could you point a package that has this behavior?  I believe the Git version of oasis2opam generates separate constraints (separate lines).

**Comment by Chris00** [comment]
@edwintorok anything left to do?

**Comment by edwintorok** [comment]
Ah sorry, I think there is nothing to do on the oasis2opam side. opam's behaviour was fixed on the linked upstream bug (though that fix hasn't made its way into a release yet).
I don't have a real package that would exhibit this problem, it was just a theoretical test (I manually modified an `_oasis` file to see if the constraints are generated properly).
I'll retest when a new opam version is released and reopen if I encounter problems.
